ignored-problems.txt

```diff
@@ -0,0 +1 @@
+Illegal access to package-private class com.intellij.dvcs.push.ui.CustomRenderedTreeNode
```

Comment:
Let's add `com.vk.admstorm:2.0.6:` to the start

Reply:
Done